packages/core-mobile/app/swap/getSwapRate.ts

```diff
@@ -80,8 +79,8 @@ export async function getSwapRate({
     }
   }
 
-  if ((priceResponse as APIError).message) {
-    return { error: (priceResponse as APIError).message }
+  if ((priceResponse as Error).message) {
+    return { error: (priceResponse as Error).message }
   }
```

Comment:
> I compared the getRate functions between the old and new sdk, and it seems not to return Error anymore. 

if this is true, then I guess we can make `swapService.getSwapRate` function not to return error with `priceResponse` object.

Reply:
update [here](3f8a29109e9e02dfb3c3e6ad6e4210d4fe81c60c)